### PR TITLE
Truncate spec|status-val postfix

### LINKED ISSUE #421: Rename `spec-val` and `status-val` of the Diff
We can simply drop the `-val` suffix. It could be retrofitted in the part which converts between clojure and js data-structures (no need to modify the differ logic itself)

### PR DESCRIPTION
**Pull request by Emulebest**
resolves #421

### CODE REVIEW

File: papiea-engine/__tests__/task_tests/intentful_task.test.ts [hunk 1]
```diff
@@ -136,8 +136,8 @@ describe("Intentful Task tests", () => {
             }
         })
         expect(task.status).toEqual(IntentfulStatus.Pending)
-        expect(task.diffs[0].diff_fields[0]["spec-val"][0]).toEqual(20)
-        expect(task.diffs[0].diff_fields[0]["status-val"][0]).toEqual(null)
+        expect(task.diffs[0].diff_fields[0]["spec"][0]).toEqual(20)
+        expect(task.diffs[0].diff_fields[0]["status"][0]).toEqual(null)
```
Comment (shlomiv) [inline papiea-engine/__tests__/task_tests/intentful_task.test.ts:140]: You could change one of these to `.spec` notation to make sure it works too

File: papiea-engine/papiea-lib-clj/src/papiea-lib-clj/core.cljs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 (ns papiea-lib-clj.core
   (:require [cljs.nodejs :as nodejs]
-            [instaparse.core :as insta]))
+            [instaparse.core :as insta])
+  (:use [clojure.set :only (rename-keys)]))
```
Comment (shlomiv) [inline papiea-engine/papiea-lib-clj/src/papiea-lib-clj/core.cljs:4]: Don't use `use`, instead do `:refer [rename-keys]` in `:require`

**Comment by shlomiv** [comment]
Looks good, small comments

File: papiea-engine/papiea-lib-clj/src/papiea-lib-clj/core.cljs (hunk 2 above)
Comment (Emulebest) [reply papiea-engine/papiea-lib-clj/src/papiea-lib-clj/core.cljs:4]: I will change that, just out of curiosity why do we not use `use`?

**Pull request merged**